Dockerfile

```diff
@@ -1,21 +1,24 @@
-FROM node:8.12.0-slim
-
-RUN mkdir -p /usr/src/app
-WORKDIR /usr/src/app
-
-COPY package.json yarn.lock tsconfig.json tsconfig.prod.json gulpfile.js /usr/src/app/
-COPY src/main /usr/src/app/src/main
-
+# ---- Base image ----
+FROM node:8.12.0-slim as base
```

Comment:
alpine is fine to use now would be smaller but doesn't really matter

Reply:
Yup you are right Tim, though we didn't want to change too much regarding the base images as we are still assessing the best choices. As suggested by @timwebster9 some base images can be small and AKS-friendly but until then, we think it is good enough :)

Thx mate !